### ISSUE TITLE
Missing playground & tracing of GqlModuleOptions in version 7.11.0

### ISSUE THREAD

**Issue by faboulaws** [nestjs/graphql#1697]
## I'm submitting a...

<pre><code>
[x] Regression 
[ ] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior
I have 2 different projects using version `7.11.0` & `7.10.3`, and the  configuration below used to works for both

```typescript
 GraphQLModule.forRoot({
      autoSchemaFile: join(process.cwd(), 'src/schema.gql'),
      sortSchema: true,
      playground: !isProduction(),
      tracing: !isProduction()
    })
```

It recently stopped working with version  `7.11.0`. with the error below
```
Argument of type '{ autoSchemaFile: string; sortSchema: true; playground: boolean; tracing: boolean; }' is not assignable to parameter of type 'GqlModuleOptions'.
  Object literal may only specify known properties, and 'playground' does not exist in type 'GqlModuleOptions'.

25       playground: !isProduction(),
         ~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
```

When comparing the two projects using `npm ls` i noticed this 

Project 1
```
>npm ls  apollo-server-core
├─┬ @nestjs/graphql@7.10.3
│ ├─┬ @apollo/gateway@0.24.4
│ │ └── apollo-server-core@2.22.2  deduped
│ └─┬ apollo-server-testing@2.22.2
│   └── apollo-server-core@2.22.2  deduped


```

Project 2
```
>npm ls  apollo-server-core

├─┬ @nestjs/graphql@7.11.0
│ ├─┬ @apollo/gateway@0.29.1
│ │ └── apollo-server-core@3.1.2 
│ └─┬ apollo-server-testing@2.25.2
│   └── apollo-server-core@2.25.2
```

To summarise the difference, apollo-server-core@2.22.2 has the option `playground` while the apollo-server-core@3.1.2 does not. This is breaking the Config signature  between `@nestjs/graphql@7.11.0` and `@nestjs/graphql@7.10.3`

## Expected behavior
I expected no change in behaviours

## Minimal reproduction of the problem with instructions

```bash
npm install @nestjs/graphql@7.11.0
```

create a app using the GraphQLModule like below
```typescript
 GraphQLModule.forRoot({
      autoSchemaFile: join(process.cwd(), 'src/schema.gql'),
      sortSchema: true,
      playground: !isProduction(),
      tracing: !isProduction()
    })
```

## Environment

<pre><code>
Nest version: 7.5.4

 
For Tooling issues:
- Node version: v12.20.0
- Platform:  Mac

Others:

</code></pre>

**Comment by jmcdo29** [comment]
Nest gets the options for `GraphQlModule` from the `apollo-server-core` package. If v3 is installed, the configs change, because they change for `apollo-server-core` v3. This is why in the docs it mentions to install `apollo-server-express@2.x.x`.

**Comment by faboulaws** [comment]
I have `apollo-server-express` installed. 
```
> npm ls apollo-server-core      
├─┬ @nestjs/graphql@7.11.0
│ ├─┬ @apollo/gateway@0.29.1
│ │ └── apollo-server-core@3.1.2 
│ └─┬ apollo-server-testing@2.25.2
│   └── apollo-server-core@2.25.2 
└─┬ apollo-server-express@2.25.2
  └── apollo-server-core@2.25.2
```

**Comment by faboulaws** [comment]
I found a workaround. The order of installation is important


With 
```
npm r @nestjs/graphql
npm r apollo-server-express
npm i apollo-server-express@2.25.2
npm i @nestjs/graphql@7.11.0
```

we get 

```
npm ls apollo-server-core         
├─┬ @nestjs/graphql@7.11.0
│ ├─┬ @apollo/gateway@0.29.1
│ │ └── apollo-server-core@2.25.2  deduped
│ └─┬ apollo-server-testing@2.25.2
│   └── apollo-server-core@2.25.2  deduped
└─┬ apollo-server-express@2.25.2
  └── apollo-server-core@2.25.2 
```

With
```
npm r @nestjs/graphql
npm r apollo-server-express
npm i @nestjs/graphql@7.11.0
npm i apollo-server-express@2.25.2
```

we get 

```
 npm ls apollo-server-core
├─┬ @nestjs/graphql@7.11.0
│ ├─┬ @apollo/gateway@0.29.1
│ │ └── apollo-server-core@3.1.2 
│ └─┬ apollo-server-testing@2.25.2
│   └── apollo-server-core@2.25.2 
└─┬ apollo-server-express@2.25.2
  └── apollo-server-core@2.25.2
```


The order of installation fixes the problem.
